### ISSUE TITLE
backtick spacing glitch in Nunito web fonts

### ISSUE THREAD

**Issue by wolftune** [google/fonts#1527]
` character seems to have no space given to it for some reason.

Example from https://community.snowdrift.coop/t/why-stack-build-ghc-mod/690

![nunito bug](https://user-images.githubusercontent.com/2801853/38464140-427c3f20-3abd-11e8-9f62-1225dea36c47.png)

We explored several options. A thought was that woff file was problematic and TT was working, but an attempt to force TT for desktop browsers seems to have not fixed it. Reportedly the glitch was not seen by some on mobile devices.

To be clear, in the case above, we're hosting the font files and serving directly as opposed to using a request to Google Fonts.

**Comment by m4rc1e** [comment]
Looks just fine to me.

![screen shot 2018-04-09 at 09 16 57](https://user-images.githubusercontent.com/7525512/38487032-c78c8b10-3bd6-11e8-9979-ca901a500319.png)


![screen shot 2018-04-09 at 09 13 07](https://user-images.githubusercontent.com/7525512/38486893-45741986-3bd6-11e8-9803-b589fdea5d13.png)

https://jsbin.com/yadoqefeli/edit?html,output

Which OS/browser are you using? do you have a copy installed locally?

**Comment by wolftune** [comment]
Interesting. I'm not the only one seeing the bug, but anyway, I'm on KDE Neon (i.e. Ubuntu 16.04). I see the bug in both Firefox and Chrome. I did have a copy of Nunito installed locally, but I tried removing it and restarting multiple browsers and the bug is still there. It seems unrelated to the local install.

I should note that I only see the bug in the title on that page and not in smaller fonts on the same page (such as when editing the post).

**Comment by wolftune** [comment]
Update: forcing TTF instead of woff2 *does* fix the issue for me in 400 nunito-regular (i.e. TTF does *not* show the bug), but it does not fix the bug for the larger title).

To clarify: I can get the bug *both* in the larger title and in regular paragraph text. Forcing Truetype solves the bug for the smaller paragraph text only.

**Comment by wolftune** [comment]
Yeah, this is weird. I just tested in LibreOffice, no problems there. I see the problem in my Firefox tab name for the link posted originally. When that tab is active, it's not an issue in the window title. It *is* a problem in the hover pop-up when hovering over the tab. Whatever is going on, I dunno. Any ideas where else to report this?

**Comment by fitojb** [comment]
bugzilla.mozilla.org

**Comment by wolftune** [comment]
Hrm, I found this closed issue. Linking here so things get seen if people notice this again: https://bugzilla.mozilla.org/show_bug.cgi?id=1334792

I don't know what to do next, but thanks for helping debug this.
